app/styles/main.scss

```diff
@@ -76,12 +76,10 @@ p {
   margin-right: 10px;
 }
 
-.ng-enter {
+.ng-enter.enter {
```

Comment:
this filters which elements are animated using a translate x animation. I would therefore call this: `.animate-slide-in-right`